analysis/tests/data_structures_test.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2022 OpenMined.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data structures Test"""
+from absl.testing import absltest
+from absl.testing import parameterized
+
+import analysis
+import pipeline_dp
+
+
+class MultiParameterConfiguration(parameterized.TestCase):
```

Comment:
For reviewer: These tests were moved from utility_analysis_tests and they were extended by adding test for noise_kind and partition_selection_strategy